tests/js/common/test-data/search/docs/generate_ii_sa_dataset.py

```diff
@@ -102,52 +115,36 @@ def main():
 
     docs = []
     collectionName = "docCollection"
-    numOfDocs = 100
+    numOfDocs = 150
     for i in range(numOfDocs):
-        key = str(collectionName) + "-" + str(i)
+        key = str(i + 100)
         ids.append(collectionName + "/" + key)
         doc = {
             "_key": key,
-            "Familie": {},
-            "Setzt": [],
-            "Personal": {
-                "ids": {
-                    "_system": {
-                        "magische Zahl": []
-                    }
-                }
-            },
-            "Haustiere": []
+            "Familie": {
+                "Mutter": {}
+            }
         }
         
-        members = random.choices(Familienmitglieder, k=random.randrange(0, len(Familienmitglieder)))
-        for m in members:
-            doc["Familie"][m] = generatePersonalData()
-        numOfSetzt = random.randint(0, 6)
-        for _ in range(numOfSetzt):
-           doc["Setzt"].append(generatePlaces())
-        
-        doc["Personal"]["ids"]["_system"]["magische Zahl"] = np.random.randint(-100,100, random.randint(0, 7)).tolist()
-        for _ in range(0, random.randint(0, 6)):
-            doc["Haustiere"].append(generateHaustiere())
+        doc["Familie"]["Mutter"] =  generatePersonalData()
         docs.append(doc)
 
     resultCollection[collectionName] = docs
 
-    edges = []
-    collectionName = "edgeCollection"
-    numOfEdges = 300    
-    for i in range(numOfEdges):
-        _from, _to =  random.choices(ids, k=2)
-        edges.append(
-            {
-                "_key": str(i),
-                "_from": _from,
-                "_to": _to,
-                "Personal": generatePersonalData()
-            }
-        )
-    resultCollection[collectionName] = edges
+    # edges = []
+    # collectionName = "edgeCollection"
+    # numOfEdges = 300    
+    # for i in range(numOfEdges):
+    #     _from, _to =  random.choices(ids, k=2)
+    #     edges.append(
+    #         {
+    #             "_key": str(i),
+    #             "_from": _from,
+    #             "_to": _to,
+    #             "Personal": generatePersonalData()
+    #         }
+    #     )
+    # resultCollection[collectionName] = edges
```

Comment:
Is it correct?

Reply:
Thanks for noticing. This should be reverted